app/admin/RTMediaUploadTermsAdmin.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * RTMediaUploadTermsAdmin.php
+ * This file sets the admin settings and handle feature of Upload Terms at admin side.
+ *
+ * @package rtMedia
+ * @author  Malav Vasita <malav.vasita@rtcamp.com>
+ */
+
+/**
+ *  Class for Admin settings regarding Upload Terms
+ */
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+if ( ! class_exists( 'RTMediaUploadTermsAdmin' ) && ! is_plugin_active( 'rtmedia-upload-terms/index.php' ) ) {
+	class RTMediaUploadTermsAdmin {
+
+		/**
+		 * Message for lable on front end side.
+		 *
+		 * @var string
+		 */
+		public $upload_terms_message;
+		/**
+		 * Error message for lable on front end side.
+		 *
+		 * @var string
+		 */
+		public $upload_terms_error_message;
+
+		/**
+		 * Constructing settings for upload terms.
+		 */
+		public function __construct() {
+			$this->upload_terms_message       = esc_html__( 'terms of services.', 'buddypress-media' );
```

Comment:
`'terms of services.'` is this correct or `T` should be capitalized?

Reply:
this is correct. Because when we put a string like I agree to `terms and condition`. Then `t` should be small.